.travis.yml

```diff
@@ -1,12 +1,14 @@
 language: ruby
 
 rvm:
-  - 2.3.0
-  - 2.3.3
-  - 2.4.0
+  - 2.2.10
+  - 2.3.7
+  - 2.4.4
+  - 2.5.1
   - ruby-head
 
 gemfile:
+  - gemfiles/rails_40.6.gemfile
```

Comment:
I appreciate the effort of checking what the minimal version of Rails is that can be run in combination with `deffering`, but let's just remove support for Rails 4.0 and Rails 4.1 all together.

Two reasons:
- http://guides.rubyonrails.org/maintenance_policy.html#severe-security-issues  
Patches for severe security issues are no longer provided by Rails for these versions.
- http://blog.scoutapp.com/articles/2017/01/09/state-of-the-2017-rails-stack  
These versions are not being used a lot (blog was from early 2017, so probably less now).

Could you reduce the appraisals to:
- rails-42
- rails-50
- rails-51
- rails-52